src/wp-includes/script-loader.php

```diff
@@ -3640,3 +3640,19 @@ function _wp_theme_json_webfonts_handler() {
 	add_action( 'wp_enqueue_scripts', $fn_generate_and_enqueue_styles );
 	add_action( 'admin_init', $fn_generate_and_enqueue_editor_styles );
 }
+
+/**
+ * Loads classic theme styles on classic themes.
+ *
+ * This is needed for backwards compatibility for button blocks specifically.
+ */
+function gutenberg_enqueue_classic_theme_styles() {
+	if ( ! wp_is_block_theme() ) {
+		wp_register_style( 'classic-theme-styles', "/wp-includes/css/dist/block-library/classic.css", array(), true );
```

Comment:
Let's adopt the linter's suggestion here:

```suggestion
		wp_register_style( 'classic-theme-styles', '/wp-includes/css/dist/block-library/classic.css', array(), true );
```

Reply:
At second glance, do we need `-$suffix` here (in case we need RTL version)?

Reply:
Done!